### PR TITLE
landing-page: add conceptdoi

### PR DESCRIPTION
**Pull request by zzacharo**
Screenshot 2023-07-21 at 18 02 52

### CODE REVIEW

File: site/cds_rdm/assets/semantic-ui/js/cds_rdm/src/records/detail.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * Copyright (C) 2016-2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import $ from "jquery";
+
+// Initialize conceptdoi modal
+$("#record-conceptdoi-badge").on("click", function () {
```
Comment (kpsherva) [inline site/cds_rdm/assets/semantic-ui/js/cds_rdm/src/records/detail.js:12]: shelve: ideally the event should be generic for any modal, not depending on the element ID

File: templates/semantic-ui/invenio_app_rdm/records/details/doi.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{#
+    Copyright (C) 2023 CERN.
+
+    CDS-RDM is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+#}
+
```
Comment (kpsherva) [inline templates/semantic-ui/invenio_app_rdm/records/details/doi.html:7]: minor: jinja formatting missing

**Review by kpsherva — APPROVED** [review]

File: site/cds_rdm/assets/semantic-ui/js/cds_rdm/src/records/detail.js (hunk 1 above)
Comment (ntarocco) [reply site/cds_rdm/assets/semantic-ui/js/cds_rdm/src/records/detail.js:12]: I am not sure that I understand your comment, we want to open that exact modal, no?

**Review by ntarocco — APPROVED** [review]
👏🏼

File: site/cds_rdm/assets/semantic-ui/js/cds_rdm/src/records/detail.js (hunk 1 above)
Comment (kpsherva) [reply site/cds_rdm/assets/semantic-ui/js/cds_rdm/src/records/detail.js:12]: yes, only the one specific modal should be opened, but it is possible to implement in a generic way (if not already done in the plain semantic)

**Pull request merged**